slate-video-library/src/Pagestructure.js

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import App from "./App";
 import ExplorepageContext from "./Components/Context/ExplorepageContext";
+import SingleVideopageContext from "./Components/Context/SingleVideopageContext";
 
 function Pagestructure() {
   return (
     <div>
       <ExplorepageContext>
-        <App />
+        <SingleVideopageContext>
```

Comment:
Suggestion: Kindly check if there is still a usage of singleVideopagecontext, as you can use UseParams to send the videoId and then call the api

Reply:
yes I am working on it as I want to access the id here and later want to call my function inside singlevideocontext